src/gui/vector/qgsattributesformproperties.h

```diff
@@ -220,8 +229,16 @@ class GUI_EXPORT QgsAttributesFormBaseView : public QTreeView, protected QgsExpr
      */
     void selectFirstMatchingItem( const QgsAttributesFormData::AttributesFormItemType &itemType, const QString &itemId );
 
+    /**
+     * Sets the filter text to the underlying proxy model.
+     *
+     * \param text Filter text to be used to filter source model items.
```

Comment:
\since ?

Reply:
It's part of a new class for 3.44, which already has the `\since`.
If we push this PR and merge it before 3.44, the `\since` of the class will be enough.